### PR TITLE
Added follow ups on inquiry summary

### PR DESCRIPTION
**Pull request by keshavgr8**

### CODE REVIEW

**Review by aayushKhandpur — CHANGES_REQUESTED** [review]
Address reviews

File: client-src/src/pages/counselor/inq-summary/inq-summary.html [hunk 1]
```diff
@@ -69,19 +69,20 @@ <h2><b>Reminders: </b></h2>
...
             </ion-row>
           </ion-item-divider>
           <ion-card-content>
             <br>
-            <p><b>State:</b>&nbsp;Lorem ipsum dolor sit amet</p>
-            <p><b>Remark:</b>&nbsp;Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
-            <p><b>Next Follow Up On:</b>&nbsp;10/01/2018 @ 3 PM</p>
+            <p><b>Status:</b>&nbsp;{{followUp?.followUpStatus}}</p>
+            <p><b>Sub Status:</b>&nbsp;{{followUp?.subStatus}}</p>
+            <p><b>Remark:</b>&nbsp;{{followUp?.remark}}</p>
```
Comment (aayushKhandpur) [inline client-src/src/pages/counselor/inq-summary/inq-summary.html:87]: Is remark inside of the follow up loop?

File: client-src/src/pages/counselor/inq-summary/inq-summary.html [hunk 2]
```diff
@@ -69,19 +69,20 @@ <h2><b>Reminders: </b></h2>
       </ion-row>
     </ion-item-divider>
     <ion-card-content>
-      <ion-card>
+      <ion-card *ngFor="let followUp of currentInqFollowUps; let i = index;">
```
Comment (aayushKhandpur) [inline client-src/src/pages/counselor/inq-summary/inq-summary.html:75]: Is this part of a sub-section?

File: client-src/src/pages/counselor/inq-summary/inq-summary.html (hunk 1 above)
Comment (keshavgr8) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:87]: Yes.

File: client-src/src/pages/counselor/inq-summary/inq-summary.html (hunk 2 above)
Comment (keshavgr8) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:75]: I don't get your query. Can you please elaborate.

File: client-src/src/pages/counselor/inq-summary/inq-summary.html (hunk 1 above)
Comment (aayushKhandpur) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:87]: Reminder as of now is inquiry level. Not on the level of follow-up

Comment (keshavgr8) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:87]: This is the remark for the follow up, how is this relevant to the reminder, i am a bit confused.

Comment (aayushKhandpur) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:87]: Ok. I got confused

File: client-src/src/pages/counselor/inq-summary/inq-summary.html (hunk 2 above)
Comment (aayushKhandpur) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:75]: Attach UI screenshot for clarity

Comment (keshavgr8) [reply client-src/src/pages/counselor/inq-summary/inq-summary.html:75]: inquiry summary page

**Review by aayushKhandpur — APPROVED** [review]

**Pull request merged**